docs/sample.yml

```diff
@@ -376,6 +445,37 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+  
+  #Tak
+  /courses/{id}/user:
+    get:
+      tags:
+        - courses
+      description: Get all user-ids that a course has.
+      responses:
```

Comment:
need request parameters under the description like below example

      parameters:
      - name: id
        in: path
        description: Course ID
        required: true
        schema:
          type: string